templates/org_mozilla_fenix_derived/clients_last_seen_v1/init.sql

```diff
@@ -1,15 +1,12 @@
 CREATE TABLE
-  glean_clients_last_seen_raw_v1
+  `moz-fx-data-shared-prod.org_mozilla_fenix_derived.clients_last_seen_v1`
 PARTITION BY
   submission_date
 CLUSTER BY
   app_name,
   os
-  -- We are receiving errors if we include sample_id as a clustering field;
-  -- see https://enterprise.google.com/supportcenter/managecases#Case/001000000040sBR/U-19888772
-  --,sample_id
```

Comment:
This should be fixed now.

Reply:
if this is fixed should we include sample_id ?

Reply:
Yes, that was my intent. Will fix.